### ISSUE TITLE
Emoji 12.0 codepoints are missing

### ISSUE THREAD

**Issue by afmenez** [SeungheonOh/EmojiTools#2]
[Emoji 12.0](https://emojipedia.org/emoji-12.0/) codepoints are missing.

**Comment by SeungheonOh** [comment]
Yes, I planned for this currently looking for full emoji list with name with it

**Comment by afmenez** [comment]
The page linked has all the Emoji 12.0 chars with names.  Under "related" are links to all previous versions.

**Comment by SeungheonOh** [comment]
Gotcha I'll try it

**Comment by afmenez** [comment]
Great, but you won't have full support until you fix #5.  Most of the chars in [Emoji 4.0](https://emojipedia.org/emoji-4.0/), for example, use ZWJ.  You can't even have full [Emoji 2.0](https://emojipedia.org/emoji-2.0/) support without ZWJ.

**Comment by SeungheonOh** [comment]
as I said, it's possible by making map uses 64 bit int, but I think that's a waste of resources, we'll figure that out

**Comment by SeungheonOh** [comment]
maybe instead of using std::map, I could use external .txt file for that, that might work!

**Comment by SeungheonOh** [comment]
just made a new branch, by adding functionality of using external list file would help making this more easy to maintain!

**Comment by SeungheonOh** [comment]
it's 200 times slower,, would it be still okay?

**Comment by afmenez** [comment]
I guess it depends on the project using it.

**Comment by SeungheonOh** [comment]
Almost done!
